app/views/shared/_topbanner.html.erb

```diff
@@ -1,7 +1,8 @@
 <section>
   <div class="container-fluid">
-    <div class="jumbotron" align="middle">
-      <img src="../../assets/images/top-banner-logo.png" height="125" alt="Top banner Taste of Code" />
+    <div class="container" align="middle">
```

Comment:
As per the Bootstrap docs, [containers are not nestable](http://getbootstrap.com/css/#overview-container). Please remove one of them.